lib/dash/dash_parser.js

```diff
@@ -122,6 +123,14 @@ shaka.dash.DashParser = class {
 
     /** @private {shaka.util.Timer} */
     this.updateTimer_ = new shaka.util.Timer(() => {
+      if (this.mediaElement_ && !this.config_.continueLoadingWhenPaused) {
+        this.eventManager_.unlisten(this.mediaElement_, 'timeupdate');
+        if (this.mediaElement_.paused) {
+          this.eventManager_.listenOnce(
+              this.mediaElement_, 'timeupdate', () => this.onUpdate_());
```

Comment:
Question(non-blocking): Should this listen for playing event instead? Since, timeupdate could be called as part of a seek, and we may still be paused at that point. I guess it would eventually get back here and stop reloading if we're still paused?

Reply:
The use of timeupdate is intended to detect the seek :) When you do seek even if you are paused, the player downloads the segments of that position, so it is necessary to detect this case